gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/ConsultationListMapperTest.java

```diff
@@ -134,6 +144,7 @@ public void testValidFullDataFlatTopicList() {
 
     @Test
     public void testValidFullDataCategoryList() {
+        var ehrExtract = unmarshallEhrExtractElement(FULL_VALID_CONSULTATION_LIST_XML);
```

Comment:
I'm confused by this new variable used throughout these tests, is it used by anything?

Reply:
Not used anywhere so removed.